bundles.config.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+/* global exports */
+
+exports.config = [
+  {'name': 'amp-3d-gltf', 'version': '0.1'},
+  {'name': 'amp-3q-player', 'version': '0.1'},
+  {'name': 'amp-access', 'version': '0.1', 'options': {'hasCss': true}},
+  {
+    'name': 'amp-access-laterpay',
+    'version': '0.1',
```

Comment:
Nit: These two versions can be combined:

`'version': ['0.1', '0.2']`

Reply:
done.